commands.js

```diff
@@ -1780,10 +1780,19 @@ var commands = exports.commands = {
 		room.decision(user, 'choose', 'team ' + target);
 	},
 
-	addplayer: function (target, room, user) {
+	addplayer: function	(target, room, user) {
```

Comment:
This sort of thing can be caught by looking over the diff (either in the GUI Git client or by using `git diff`), which you should always do before committing so you know exactly what you're committing.